src/rdkafka.h

```diff
@@ -3674,6 +3674,17 @@ rd_kafka_assignment (rd_kafka_t *rk,
                      rd_kafka_topic_partition_list_t **partitions);
 
 
+/**
+ * @brief Check whether the consumer considers the current assignment to
+ *        have been lost involuntarily. This method is only useful when
+ *        called from within a group rebalance callback (rebalance_cb)
```

Comment:
"only useful on a subscribing high-level consumer instance when .."

`@remark Calling rd_kafka_assign() resets this flag`  - or something along those lines.
This is important to state so that people dont do:
```c
    assign(NULL);
    if (assignment_lost())
        yada;
```

Reply:
Also: "a group rebalance callback... or RD_KAFKA_EVENT_REBALANCE .."

Reply:
good points.